Formula/urler.rb

```diff
@@ -0,0 +1,21 @@
+class Urler < Formula
+  desc "Command-line tool for URL parsing and manipulation"
+  homepage "https://github.com/curl/urler"
+  url "https://github.com/curl/urler/archive/64a48c004cbc0e91d15aff2e090f423ab7555b6c.tar.gz"
+  version "0.1"
+  sha256 "730d4d2611d1844a0389b71a214bd8b0b1c86724f65ec913d777af482a7db496"
+  license "curl"
+
+  def install
+    system "make"
+    bin.install "urler"
+    man1.install "urler.1"
```

Comment:
```suggestion
    system "make", "BINDIR=#{bin}", "MANDIR=#{man}", "install"
```

Reply:
Created an upstream PR to add support for this https://github.com/curl/urler/pull/17